subprojects/docs/src/docs/userguide/kotlin_dsl.adoc

```diff
@@ -131,6 +132,35 @@ image::intellij-script-dependencies-reload.png[Reload script dependencies]
 It is recommended to *disable automatic build import* but *enable automatic reload of script dependencies.*
 That way you get early feedback while editing Gradle scripts while controlling when the whole build setup gets synchronized with your IDE.
 
+=== Troubleshooting
+
+The IDE support is provided by two components:
+
+* the IntelliJ IDEA or Android Studio Kotlin Plugin version,
+* the version of Gradle used to import the build.
+
+In you run into trouble, the first thing to try is to run `gradle tasks` to see if your issue is IDE related only and to possibly get more details.
```

Comment:
❓ Suggest using `gradle tasks --stacktrace`?

Reply:
`gradle tasks` should be enough in most cases, and it will suggest running with `--stacktrace`